src/App.js

```diff
@@ -4,16 +4,30 @@ import { CreateClassForm } from 'modules/CreateClassForm';
 import { Preview } from 'modules/Preview';
 import SpikeSheet from './components/SpikeSheet';
 import './App.css';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  display: grid;
+  grid-template-columns: 1fr minmax(auto, 600px) 1fr;
+  grid-template-rows: 25% auto;
+  grid-template-areas:
+  	"header header header"
+	". content .";
+`;
+
+const Content = styled.div`grid-area: content;`;
 
 class App extends Component {
   render() {
     return (
-      <div className="App">
+      <Wrapper>
         <AppHeader />
-        <CreateClassForm />
-        <Preview />
-        <SpikeSheet />
-      </div>
+		<Content>
+          <CreateClassForm />
+          <Preview />
+          <SpikeSheet />
+	    </Content>
+	  </Wrapper>
     );
```

Comment:
Mixing tabs and spaces and the indentation above is off. If you'd like help getting your linter working in Atom, I'm totally available for that.